packages/backend/src/config/config.local.ts

```diff
@@ -25,17 +28,30 @@ export function getLocalConfig(env: Env): Config {
       startedAt: new Date().toISOString(),
       commitSha: getGitCommitSha(),
     },
-    ethereumDiscovery: {
-      // This is an arbitrary number so we do not start too far in the past.
-      startBlock: env.integer('START_BLOCK', 18127698),
-      clockIntervalMs: env.integer('CLOCK_INTERVAL_MS', 10 * 1000),
-      rpcUrl: env.string('ETHEREUM_RPC_URL'),
-      etherscanApiUrl: 'https://api.etherscan.io/api',
-      etherscanApiKey: env.string('ETHERSCAN_API_KEY'),
-      etherscanMinTimestamp: new UnixTime(
-        env.integer('ETHERSCAN_MIN_TIMESTAMP', 0),
-      ),
-      discovery: discoveryConfig,
+    discovery: {
+      ethereum: {
+        // This is an arbitrary number so we do not start too far in the past.
+        startBlock: env.integer('START_BLOCK', 18127698),
+        clockIntervalMs: env.integer('CLOCK_INTERVAL_MS', 10 * 1000),
+        rpcUrl: env.string('ETHEREUM_RPC_URL'),
+        blockExplorerApiUrl: 'https://api.etherscan.io/api',
+        blockExplorerApiKey: env.string('ETHERSCAN_API_KEY'),
+        blockExplorerMinTimestamp: new UnixTime(
+          env.integer('ETHERSCAN_MIN_TIMESTAMP', 0),
+        ),
+        discovery: ethereumDiscoveryConfig,
+      },
+      arbitrum: {
+        startBlock: env.integer('START_BLOCK', 133212747),
+        clockIntervalMs: env.integer('CLOCK_INTERVAL_MS', 10 * 1000),
+        rpcUrl: env.string('ARBITRUM_RPC_URL'),
+        blockExplorerApiUrl: 'https://api.arbiscan.io/api',
+        blockExplorerApiKey: env.string('ARBISCAN_API_KEY'),
+        blockExplorerMinTimestamp: new UnixTime(
+          env.integer('ARBISCAN_MIN_TIMESTAMP', 0),
+        ),
```

Comment:
Arbiscan is unreliable on the first arbitrum blocks so we can add this (not necessary now, but will be useful if we want to download the history)
```suggestion
        // ~ Timestamp of block number 0 on Arbitrum
        blockExplorerMinTimestamp: UnixTime.fromDate(new Date('2021-05-28T22:15:00Z')),
```

Reply:
[chore: fix configuration & inteface naming](https://github.com/l2beat/lz-monitoring/pull/16/commits/87f49ca01c29da3a02163278563be390c7145f73)